Autoload.php

```diff
@@ -121,10 +121,36 @@ public function findFile($class)
                 return $dir . DIRECTORY_SEPARATOR . $classFile;
             }
         }
+
+        if ($resolvedPath = $this->findComposerMagentoModuleFile($class)) {
```

Comment:
Would it be nicer to simply do:
```php
return $this->findComposerMagentoModuleFile($class);
```
which would remove an if-statement and reduce complexity a bit?